samples/bluetooth/subrating/README.rst

```diff
@@ -0,0 +1,225 @@
+A subrated connection can therefore have a short ACL connection interval but still exhibit a low duty cycle so that power is conserved.
+
+When a higher bandwidth is needed, the subrate request procedure allows connection parameter updates to be made with minimal delay.
+
+This sample transmits data in bursts between two development kits to measure the transmission latency using different subrating parameters.
+
+To measure the transmission latency from application layers, a GATT Latency service `BT_UUID_LATENCY` is included to compute the time spent.
+When the sender writes its timestamp to the `BT_UUID_LATENC_CHAR` characteristic of the receiver, the Latency service of the receiver will automatically reply back.
+Whenever the sender receives a response, it will use its current time and the corresponding timestamp written before to estimate the round-trip time (RTT) of a writing characteristic procedure (see `Bluetooth Core Specification`_: Vol 3, Part F, 3.4.5 Writing attributes).
+
+.. list-table:: GATT Attributes
+   :header-rows: 1
+
+   * - Type
+     - UUID
+     - Property
+   * - Primary service
+     - BT_UUID_LATENCY
+     - Read only
+   * - Characteristic
+     - BT_UUID_LATENCY_CHAR
+     - BT_GATT_CHRC_WRITE, BT_GATT_PERM_WRITE
+
+By default, the following connection parameters are used:
+
+.. list-table:: Default parameter values
+   :header-rows: 1
+
+   * - Parameter
+     - Value
+   * - Connection interval
+     - 8 units (10 ms)
+   * - Subrate factor
+     - 1 or 10
+   * - Continuation number
+     - 0 or 1
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/subrating`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to both development kits, test it by performing the following steps:
+
+1. Connect to both kits with a terminal emulator (for example, `nRF Connect Serial Terminal`_).
+   See :ref:`test_and_optimize` for the required settings and steps.
+#. Reset both kits.
+#. In one of the terminal emulators, type "c" to start the application on the connected board in the central role.
+#. In the other terminal emulator, type "p" to start the application in the peripheral role.
+#. Observe that the kits establish a connection.
+#. The sample will alternate between the following configurations:
+
+   - No subrating
+   - Subrating with a factor of 10, continuation number 0
+   - Subrating with a factor of 10, continuation number 1
+#. Observe the difference in the duty cycle of the connection:
+
+   - With no subrating, the link will always use a fast connection interval of 10ms.
+   - With a subrate factor of 10 and no continuation events, the link will always use a slower effective connection interval of 100ms.
+   - With a subrate factor of 10 and one continuation event, the link will use a fast connection interval of 10ms when data needs to be sent, and switch to a low effective connection interval of 100ms otherwise.
```

Comment:
```suggestion
   * With no subrating, the link will always use a fast connection interval of 10ms.
   * With a subrate factor of 10 and no continuation events, the link will always use a slower effective connection interval of 100ms.
   * With a subrate factor of 10 and one continuation event, the link will use a fast connection interval of 10ms when data needs to be sent, and switch to a low effective connection interval of 100ms otherwise.
```